src/util/user.c

```diff
@@ -312,6 +347,7 @@ static int user_compare(CRBTree *tree, void *k, CRBNode *rb) {
  * Return: 0 on success, negative error code on failure.
  */
 int user_registry_init(UserRegistry *registry,
+                       Log *log,
                        size_t n_slots,
                        const unsigned int *maxima) {
         static_assert(sizeof(*maxima) == sizeof(*registry->maxima),
```

Comment:
You sure you don't wanna assign `log` to `registry->log` here?

Reply:
Fixed.